### PR TITLE
Feature/disable eth sdk

### PR DESCRIPTION
**Pull request by noambergIL**
Should be merged after #1587

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -163,19 +163,20 @@ jobs:
...
-      - checkout
-      - run: ./.circleci/install-go.sh
-      - run: ./.circleci/retry.sh 5 go get github.com/orbs-network/go-junit-report
-      - run: ./docker/test/ganache-related-test.sh
-      - store_artifacts:
-          path: _out
-          destination: test-output
-      - store_test_results:
-          path: _reports
+# TODO Ethereum disconnect
```
Comment (ronnno) [inline .circleci/config.yml:166]: If you must keep the comments here (i would delete them) please give a proper explanation at the top as for what it's there for and when to uncomment it. for example: `uncomment these lines to enable tests which require ganache`

File: .circleci/config.yml [hunk 2]
```diff
@@ -201,10 +202,11 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-      - ganache_related_tests:
-          filters:
-            tags:
-              only: /^v.*/
+# TODO Ethereum disconnect
```
Comment (ronnno) [inline .circleci/config.yml:205]: same

File: services/virtualmachine/sdk_ethereum.go [hunk 3]
```diff
@@ -85,6 +85,7 @@ func (s *service) handleSdkEthereumCall(ctx context.Context, executionContext *e
 // inputArg4: ethereumABIPackedInputArguments ([]byte)
 // outputArg0: ethereumABIPackedOutput ([]byte)
 func (s *service) handleSdkEthereumCallMethod(ctx context.Context, executionContext *executionContext, args []*protocol.Argument, permissionScope protocol.ExecutionPermissionScope) ([]byte, error) {
+	panic("Ethereum SDK Disabled") // TODO Ethereum Disconnect
```
Comment (ronnno) [inline services/virtualmachine/sdk_ethereum.go:88]: this is not a TODO anymore.. you can write `// ethereum access API suspended` or something. at best its a `// TODO re-enable Ethereum access (with refTime based finality)` or something

**Comment by noambergIL** [comment]
re doing PR

**Pull request closed without merging**